src/Microsoft.Azure.WebJobs.Host/Indexers/DefaultBindingProvider.cs

```diff
@@ -26,7 +25,11 @@ internal static class DefaultBindingProvider
             IExtensionRegistry extensions)            
         {
             List<IBindingProvider> innerProviders = new List<IBindingProvider>();
-            var ruleQueueOutput = new QueueAttributeBindingProvider(nameResolver, storageAccountProvider, messageEnqueuedWatcherGetter);
+
+            // Wire up new bindings 
+            IConverterManager converterManager = new ConverterManager();
+            var queueHelper = new QueueBindingRules(storageAccountProvider, messageEnqueuedWatcherGetter);
```

Comment:
I think it would look more natural to write:

var queueRules = new QueueBindingRules(...);
var queueBindingProvider = queueRules.Build(...);

Or maybe even:

var queueBindingProvider = QueueBindingRules.Build(...);

The point is to reduce the names/concepts in the mix.